File: ITP1/2/a.hs
import Control.Applicative


main = do
  [a, b] <- map ( read :: String -> Int ) . words <$> getLine
  let rel | a<b = "<"
          | a>b = ">"
          | otherwise = "=="
  putStrLn $ "a " ++ rel ++ " b"

File: ITP1/2/a.cpp
#include <iostream>
using namespace std;

int main(){
    int a, b;
    string rel;
    cin >> a >> b;
    if (a<b) {
        rel = "<";
    } else if (a>b) {
        rel = ">";
    } else {
        rel = "==";
    }
    cout << "a " << rel << " b" << endl;
    return 0;
}
